lms/models/h_group.py

```diff
@@ -4,6 +4,7 @@
 class HGroup(NamedTuple):
     name: str
     authority_provided_id: str
+    type: str = "course_group"
```

Comment:
How do we feel about `enum`s for this sort of thing?

Reply:
Not sure really. In this case it's probably easier not to bother